spec.md

```diff
@@ -633,7 +633,7 @@ public function process(string $algorithm, &...$parts)
 
 ### 4.6 Modifier Keywords
 
-Properties and methods of a class have numerous keyword modifiers that alter how the
+Classes, properties, and methods have numerous keyword modifiers that alter how the
```

Comment:
```suggestion
Classes, properties and methods have numerous keyword modifiers that alter how the
```

Reply:
The Oxford Comma is the correct way to write English, and omitting is is an error.